kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -13,12 +14,7 @@
         <h1 :style="{ marginLeft: '-8px' }">
           {{ injectedtr('otherLibraries') }}
         </h1>
-      </KGridItem>
-      <KGridItem
```

Comment:
Removing the usage of `KGridItem` here or the `margin-left: -190px;` on line 322 might have had unwanted effects on the icon and string shown when searching for other libraries:

![searchingLibraries](https://github.com/user-attachments/assets/7e1abae3-08a3-49e8-becc-5889e1004754)


I also noticed that when the window is large, the `No other libraries around you right now` and `Showing all available libraries around you` descriptions are no longer positioned underneath the `Other Libraries` section header:

![noOtherLibraries](https://github.com/user-attachments/assets/b3b3b593-c11e-4bd3-9ca1-dac7bd90318b)
![availableLibraries](https://github.com/user-attachments/assets/d08a8be1-6df5-4a15-b98e-bf8466099fba)

I think any changes you make to the connection state when there are no libraries should also be compared with the other two connection state descriptions to make sure there are no regressions.